src/js/disability-benefits/reducers/claim-sync.js

```diff
@@ -0,0 +1,39 @@
+import _ from 'lodash/fp';
+
+import {
+  SET_CLAIM_DETAIL,
+  SET_CLAIMS,
+  SET_UNAVAILABLE
+} from '../actions';
+
+const initialState = {
+  synced: true,
+  syncedDate: null,
+  available: true
+};
+
+export default function claimDetailReducer(state = initialState, action) {
+  switch (action.type) {
+    case SET_CLAIM_DETAIL:
+      return _.merge(state, {
+        synced: action.claim.attributes.successfulSync,
+        syncedDate: action.claim.attributes.updatedAt,
+        available: true
+      });
+    case SET_CLAIMS:
+      if (action.claims.length) {
```

Comment:
If they don't have any claims, we don't want to show either of these messages. They'll get a message on the claims list page if their list is empty, which is handled on the list page.